torch/csrc/autograd/python_torch_functions_manual.cpp

```diff
@@ -663,6 +663,28 @@ THPVariable__functionalize_are_all_mutations_hidden_from_autograd(
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject*
+THPVariable__functionalize_are_all_mutations_under_no_grad_or_inference_mode(
+    PyObject* self,
+    PyObject* args,
+    PyObject* kwargs) {
+  HANDLE_TH_ERRORS
+  static PythonArgParser parser(
+      {"_functionalize_are_all_mutations_under_no_grad_or_inference_mode(Tensor t)"},
```

Comment:
It's easier to pybind these in e.g. https://github.com/pytorch/pytorch/blob/main/torch/csrc/utils/python_dispatch.cpp#L738 than to manually write bindings like this.

Reply:
you're right - at some point I should move all of these functionalization bindings to pybind